### ISSUE TITLE
Can we use this repo to post "tips and tricks"?

### ISSUE THREAD

**Issue by sgaichas** [NOAA-EDAB/seaside#34]
I keep going back to old google chats to, for example, remember how I set up my 2 factor authentication on github. I've also recently sent this info to a couple of other people.

Could we have either wiki pages here or a docs folder with more in depth "how to" vignettes that are likely of interest to many of us? Not everyone is on all of the google chat rooms and not all of them are permanent.

Also, we could link to other resources like [Ben Best's R3](https://noaa-iea.github.io/r3-train/) course materials and [Eli Holmes' R workflows](https://github.com/RVerse-Tutorials) course materials from either the wiki or some docs.

**Comment by slucey** [comment]
I like the idea of setting up a docs folder to hold some "How To" vignettes.

**Comment by kimberly-bastille** [comment]
We have a wiki for this repo that includes a [resources tab](https://github.com/NOAA-EDAB/seaside/wiki/Resources). We can definitely expand that to include more topics.

**Comment by sgaichas** [comment]
Awesome @kimberly-bastille, thanks. If it is ok for me to edit directly, can add the 2 factor setup under the resources tab in the github section, and the links to training courses under a new section?

**Comment by kimberly-bastille** [comment]
@sgaichas yes please feel free to edit/add whenever you come across something that might be helpful.
